drizzlepac/haputils/photometry_graphics.py

```diff
@@ -63,21 +75,20 @@ def compute_global_stats(value_array):
     return mean_value
 
 
-def load_data_to_arrays(csv_filename):
-    """Load the harvested data, stored in a CSV file, into local arrays.
+def get_data(storage_filename):
+    """Load the harvested data, stored in a storage file, into local arrays.
 
     Parameters
     ==========
-    csv_filename: str
-    Name of the CSV file created by generate_photometry_df.
+    storage_filename: str
+    Name of the storage file for the Pandas dataframe created by the harvester.
 
     Returns
     =======
-    (d_MagAp1_mean, d_MagAp1_std, d_MagAp1_median): tuple
-    Aperture 1 statistics
-
-    (d_MagAp2_mean, d_MagAp2_std, d_MagAp2_median): tuple
-    Aperture 2 statistics
+    phot_data: Pandas dataframe
```

Comment:
```suggestion
    phot_data : Pandas dataframe
```
Another minor docstring cleanup...